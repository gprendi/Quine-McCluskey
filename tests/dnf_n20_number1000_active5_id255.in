d 20
00000010000010010110
00000010000000010100
00000000000000000110
00000000000000000100
00000000000000010110
00000000000010010100
00000010000010000110
00000010000000000000
00000010000000000010
00000010000010010100
00000010000010000100
00000000000010010000
00000010000010000000
00000010000010000010
00000000000010000010
00000000000010010110
00000000000000000000
00000000000000010010
00000000000010010010
00000000000000000010
00000000000000010100
00000000000000010000
00000000000010000100
00000010000000010110
00000010000010010000
00000010000000000100
00000010000010010010
00000010000000010010
00000000000010000110
00000000000010000000
00000010000000010000
00000010000000000110
