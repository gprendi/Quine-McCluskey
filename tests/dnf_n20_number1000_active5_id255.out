p dnf 20 0
-1 -2 -3 -4 -5 -6 -8 -9 -10 -11 -12 -14 -15 -17 -20 0
